gateway-js/src/config.ts

```diff
@@ -131,8 +131,8 @@ interface GatewayConfigBase {
 export interface RemoteGatewayConfig extends GatewayConfigBase {
   serviceList: ServiceEndpointDefinition[];
   introspectionHeaders?:
-    | HeadersInit
-    | ((service: ServiceEndpointDefinition) => Promise<HeadersInit> | HeadersInit);
+  | HeadersInit
```

Comment:
Maybe the same rule as before, but I think the indentation here is much nicer. Also, just my preference.

Reply:
Was*